dissect/target/tools/shell.py

```diff
@@ -469,6 +471,11 @@ def __init__(self, target: Target):
         self.cwd = None
         self.chdir("/")
 
+        runcommands_file = pathlib.Path(
+            getattr(target._config, "TARGETRCFILE", self.DEFAULT_RUNCOMMANDSFILE)
+        ).expanduser()
+        self.load_runcommands(runcommands_file)
```

Comment:
Why not in `TargetCmd`?

Reply:
The registry shell inherits from TargetCmd. Further, the "ordinary" shell and the registry shell are not fully compatible. Therefore, I don't think it makes sense to execute the same rc file in both shells.

We could support the registry shell with a different rc file. (I asked the team this on Thursday, and they indicated to restrict the rc file to the "ordinary" shell.)

Reply:
@JSCU-CNI would you mind giving your thoughts on this?